README.md

```diff
@@ -10,7 +10,7 @@ With the tools in the GrimoireLab toolset you can retrieve data, store it in dat
 
 ![](eclipse.png)
 
-Example dashboard produced with GrimoireLab: the [OPNFV Development Dashboard](http://opnfv.biterg.io)
+Example dashboard produced with GrimoireLab: the [OPNFV Development Dashboard](https://chaoss.biterg.io)
```

Comment:
Can you please change the link text of the dashboard?
```
- [OPNFV Development Dashboard](https://chaoss.biterg.io)
+ [CHAOSS Development Dashboard](https://chaoss.biterg.io)
```

Reply:
Okay, sure .